packages/dds/sequence/src/test/intervalTestUtils.ts

```diff
@@ -107,6 +108,25 @@ export function assertEquivalentSharedStrings(a: SharedString, b: SharedString)
 	}
 }
 
+function assertPropertiesEqual(a: SharedString, b: SharedString): void {
+	for (let i = 0; i < a.getLength(); i++) {
+		const aProps = a.getPropertiesAtPosition(i) ?? [];
+		const bProps = b.getPropertiesAtPosition(i) ?? [];
```

Comment:
```suggestion
		const aProps = a.getPropertiesAtPosition(i);
		const bProps = b.getPropertiesAtPosition(i);
```

Reply:
we shouldn't default the value here, as they should really be the same. if one is undefined, and the other is empty the check should fail

Reply:
if you need a default it should be to `{}` not `[]`